Project-12-Investment-Calculator/src/index.css

```diff
@@ -0,0 +1,96 @@
+@import url("https://fonts.googleapis.com/css2?family=Quicksand:wght@400;700&family=Roboto+Condensed:wght@400;700&display=swap");
```

Comment:
@sknepali Use CDN if possible, it's faster and effective way of prefetching fonts